pkg/controller/util/image.go

```diff
@@ -0,0 +1,41 @@
+package util
+
+import (
+	apps "k8s.io/api/apps/v1"
+
+	"github.com/openkruise/kruise/pkg/util/inplaceupdate"
+)
+
+// DiffImagesBetweenRevisions returns the different images between two revisions.
+func DiffImagesBetweenRevisions(oldRevision, newRevision *apps.ControllerRevision) map[string]string {
```

Comment:
Can you move the function to [file](https://github.com/openkruise/kruise/blob/master/pkg/control/sidecarcontrol/util.go)?

Reply:
get it

Reply:
Well, this function is also used by Clonset controller, so I think that storing it in a common directory isn't a bad idea.

Reply:
Well, I put this funtion to `pkg/controller/sidecarset/sidecarset_predownload_image.go`.

Reply:
fix it